docs/src/content/docs/fr/getting-started.mdx

```diff
@@ -85,3 +85,13 @@ Si vous oubliez quelque chose d'important, Starlight vous le fera savoir.
 Une fois que vous avez créé et personnalisé votre site Web Starlight, vous pouvez le déployer sur un serveur Web ou une plateforme d'hébergement de votre choix, y compris Netlify, Vercel, GitHub Pages et bien d'autres.
 
 [Pour en savoir plus sur le déploiement d'un site Astro, consultez la documentation Astro](https://docs.astro.build/fr/guides/deploy/)
+
+## Dépannage de Starlight
+
+Les informations [configuration de projet](/fr/reference/configuration/) et [configuration du frontmatter de la page individuelle](/fr/reference/frontmatter/) pour Starlight sont disponibles dans la section Référence de ce site. Utilisez ces pages pour vous assurer que votre site Starlight est configuré et fonctionne correctement.
+
+Utilisez la liste croissante de guides dans la barre latérale pour vous aider à ajouter du contenu et à personnaliser votre site Starlight.
+
+Si vous ne trouvez pas votre réponse dans cette documentation, veuillez consulter la [documentation complète d'Astro](https://docs.astro.build) pour obtenir la documentation complète d'Astro. Votre question peut être résolue en comprenant comment Astro fonctionne en général, sous ce thème Starlight.
```

Comment:
Looks like part of @HiDeoo’s [suggestion](https://github.com/withastro/starlight/pull/377#discussion_r1271057939) didn’t make it here:

```suggestion
Si vous ne trouvez pas votre réponse dans cette documentation, veuillez consulter la [documentation complète d'Astro](https://docs.astro.build/). Votre question peut être résolue en comprenant comment Astro fonctionne en général, sous ce thème Starlight.
```